email-ballerina/imap_listener_endpoint.bal

```diff
@@ -111,15 +101,16 @@ public class ImapListener {
             task:AppointmentConfiguration config = {cronExpression: scheduler};
             self.appointment = check new(config);
         } else {
-            task:TimerConfiguration config = {intervalInMillis: self.config.pollingIntervalInMillis, initialDelayInMillis: 100};
+            task:TimerConfiguration config =
+                {intervalInMillis: <int>(self.config.pollingInterval * 1000), initialDelayInMillis: 100};
             self.appointment = check new (config);
         }
         var appointment = self.appointment;
         if (appointment is task:Scheduler) {
             check appointment.attach(imapAppointmentService, self);
             check appointment.start();
         }
-        //log:print("User " + self.config.username.to + " is listening to remote server at " + self.config.host + "...");
+        log:print("User " + self.config.username + " is listening to remote server at " + self.config.host + "...");
```

Comment:
Can we change this like follows?
```
log:print("Listening to remote server", username = self.config.username, host = self.config.host
);

Reply:
I have tested this new way of logging. Then it seems the host is ambiguous whether it is the client's host or the server's host. If possible, it is great if we can change the color or bold some text in the logs instead which would help to highlight the host address and the username.